services/simcore/docker-compose.yml

```diff
@@ -52,7 +52,10 @@ services:
         limits:
           cpus: "0.5"
           memory: "512M"
-
+      placement:
+        constraints:
+          - node.platform.os == linux
```

Comment:
I think all the os == linux constraints you can remove. we do not have anything else than linux anyway, or otherwise put it everywhere.

Reply:
I removed these constraints